docarray/typing/tensor/audio/audio_torch_tensor.py

```diff
@@ -0,0 +1,84 @@
+import wave
+from typing import TYPE_CHECKING, BinaryIO, TypeVar, Union
+
+from docarray.typing.tensor.torch_tensor import TorchTensor, metaTorchAndNode
+
+T = TypeVar('T', bound='AudioTorchTensor')
+
+if TYPE_CHECKING:
+    from docarray.proto import NodeProto
+
+
+class AudioTorchTensor(TorchTensor, metaclass=metaTorchAndNode):
```

Comment:
remove the implementation of both methods and apply this change
```suggestion
class AudioTorchTensor(AudioTensorMixin, TorchTensor, metaclass=metaTorchAndNode):
```